AngularTask-3/src/app/pipes/greet.pipe.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { GreetPipe } from './greet.pipe';
+
+describe('GreetPipe', () => {
+  let greetPipe: GreetPipe;
+
+  beforeEach(() => {
+    greetPipe = new GreetPipe();
+  });
+
```

Comment:
At initial level its ok. Following just a suggestion. No need to update the code do as a personal exercise

But suggest you to explore all pipe combinations like built in , custom pipe, etc. 

https://medium.com/bb-tutorials-and-thoughts/angular-unit-testing-pipes-4af036a3d98d.